test/studies/shootout/mandelbrot/ldelaney/PERFEXECOPTS

```diff
@@ -0,0 +1 @@
+--n=16000
```

Comment:
I think this is a good correctness test, but I don't think it's what we want for a performance test. I think the mandelbrot test you added will be a much better for comparison.